packages/dashboard/src/views/RunsView.tsx

```diff
@@ -73,11 +75,30 @@ export function RunsView({
   if (!runFeed.runs.length) {
     return <div>No runs have started on this project.</div>;
   }
+
+  const groupedRuns: any[] = [];
+  data.runFeed.runs.forEach((run) => {
```

Comment:
What do you do if all the runs for this group are not in this page of results?

Reply:
I think this needs more thought. Particularly if we add filtering to these run results. Which I think we should.

Reply:
it will be updated after clicking `Load more`